linty_fresh/main.py

```diff
@@ -38,6 +38,8 @@ def create_parser() -> argparse.ArgumentParser:
                              'notes and filter out problems')
     parser.add_argument('files', type=str, nargs='+',
                         help='The lint file being parsed.')
+    parser.add_argument('--fail-warnings', default=True, action='store_true',
```

Comment:
I'd prefer we not modify the signature to be different for Android.  If you want, you can add `*args, **kwargs` to all of the `parse` methods so we can pass in optional arguments